sim/simulator.ts

```diff
@@ -187,7 +186,7 @@ namespace pxsim {
 
         updateStats() {
             this.stats.textContent = this.screenState.stats;
-            this.tryScreenshot();
+            // screenshots are handled in the share dialog
```

Comment:
What about save as PNG?

Reply:
It takes the screenshot when you press the button.

Reply:
Good enough until we figure out something better